pkgs/test/lib/src/runner/browser/platform.dart

```diff
@@ -126,7 +128,11 @@ class BrowserPlatform extends PlatformPlugin
   /// Mappers for Dartifying stack traces, indexed by test path.
   final _mappers = <String, StackTraceMapper>{};
 
+  /// The default template for html tests
```

Comment:
[nit] End in a period. (here and elsewhere)


https://dart.dev/guides/language/effective-dart/documentation#do-format-comments-like-sentences